README.md

```diff
@@ -240,21 +240,126 @@ Similar to providers, plugins are independent projects that conform to the Dayto
 
-Afterwards, navigate to the [contributing guide](CONTRIBUTING.md) to get started.
+  1. **Read the Developer Certificate of Origin Version 1.1** 
 
-## License
+      Please review the [Developer Certificate of Origin Version 1.1](https://developercertificate.org/) to understand the contribution requirements.
+
+  2. **Signing all commits to the Daytona project** 
+
+      Ensure that all your commits are signed to comply with the Daytona project's contribution policies.
+
+      This ensures that users, distributors, and other contributors can rely on all the software related to Daytona being contributed under the terms of the [Apache License 2.0](LICENSE). No contributions will be accepted without following this process.
+
+### Ways to Contribute
+
+### 1. Reporting Issues and Suggesting Features
+
+Creating issues is a valuable way to contribute by reporting bugs, suggesting features, or improving documentation.
+
+Before creating a new issue, search the existing issues [here](https://github.com/daytonaio/daytona/issues) to see if your concern has already been addressed.
+
+* If no existing issue matches your contribution, follow these steps:
+   1. **Identify the Type of Issue**
+      * **Bug Report:** If you encounter unexpected behavior or errors.
+      * **Feature Request:** If you have an idea for a new feature or improvement.
+      * **Documentation Improvement:** If you notice gaps or areas for improvement in the documentation.
+
+   2. **Create a new issue**
+      * Navigate to Issues: Go to the Issues tab [here](https://github.com/daytonaio/daytona/issues).
+      * Click on "New Issue": Choose the appropriate template (Bug Report, Feature Request, etc.) if available.
+      * Fill Out the Issue Template: Provide a clear and concise description of the issue, including steps to reproduce (for bugs) or detailed feature descriptions.
+      * Submit the Issue: Click "Submit new issue" to create the issue.
+   3. Engage with the Community
+      * **Respond to Feedback:** Be prepared to provide additional information or clarification if maintainers or other contributors have questions.
+      * **Collaborate on Solutions:** If you have ideas for resolving the issue, share them in the comments.
+
+### 2. Contributing Code
+
+If you're interested in contributing code to Daytona, follow these steps:
+
+1. **Fork the Daytona repository**
+
+    [Fork](https://github.com/daytonaio/daytona/fork) the GitHub repository to create your own copy of the repository.
+  
+2. **Create a Workspace with Daytona**
+
+    Use the Daytona CLI to create a workspace for your forked repository. Replace YOUR-FORK-URL with the URL of your forked repository.
+    ```bash
+    daytona create YOUR-FORK-URL
+    ```
+3.  **Create a new branch**
+    
+    Once in the development conatiner, create a new branch for your changes:
+    ```bash
+    git checkout -b my-new-feature
+    ```
+
+4. **Run the Project Locally**
```

Comment:
I would change this in this direction

4. **Running Daytona in development**
    A `dtn` alias is automatically created inside the Workspace. You can use it to compile and run daytona.
   For example:
   `dtn serve`